lib/control.js

```diff
@@ -98,8 +98,30 @@ const sequential = (fns, context, callback) => {
   next();
 };
 
+// Run `asyncFn` if `condition` is truthy, else return `defaultVal` to callback.
+//   condition - <any>,
+//   defaultVal - <any>, optional, value that will be returned to callback if
+//       `condition` is falsy.
+//   asyncFn - <Function>, callback-last function that will be executed if
+//       `condition` if truthy
+//   args - <any[]>, args to pass to `asyncFn`
+const runIf = (condition, defaultVal, asyncFn, ...args) => {
+  if (typeof defaultVal === 'function') {
+    args.unshift(asyncFn);
+    asyncFn = defaultVal;
+    defaultVal = undefined;
+  }
+  if (condition) {
+    asyncFn(...args);
+  } else {
+    const callback = common.safeCallback(args);
```

Comment:
What would be the point of calling this function without a callback?

Reply:
Maybe, synchronous function passed as `asyncFn`.

Reply:
@o-rumiantsev, in that case calling `metasync.runIf(condition, asyncFn, ...args)` can be replaced with
```js
if (condition) {
  asyncFn(...args); // synchronous function
}
```

Reply:
So, should I revert this change?

Reply:
You probably should.